src/redist/targets/packaging/windows/clisdk/generatebundle.ps1

```diff
@@ -2,6 +2,7 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 param(
+    [Parameter(Mandatory=$true)][string]$UpgradePoliciesWxsFile,
```

Comment:
Is there any chance some other product would have a dependency on this ps1 script and fail due to missing the new mandatory parameter?

Reply:
no, this is purely for building the SDK installer EXE bundle